### ISSUE TITLE
sess.run() taking much time to run

### ISSUE THREAD

**Issue by sheshantsinha** [ArifSohaib/tensorflow-sentiment-analysis-and-data-read#3]
I am running data_read script on my own my computer(RAM-8GB,CPU version). I am using this script on my own data of size(4000,14) training data and (1000,14) testing data
`raw_tweet,label=sess.run([raw_tweet_op,label_op])` taking much time and not responding.

**Comment by ArifSohaib** [comment]
Did you try using the tensorflow Dataset API?
https://www.tensorflow.org/api_docs/python/tf/data/Dataset
I haven't updated the repo in a while but I will try to convert this project over to that soon.
